docs/src/examples/components/SplitButton/Usage/index.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react'
+
+import ComponentExample from 'docs/src/components/ComponentDoc/ComponentExample'
+import ExampleSection from 'docs/src/components/ComponentDoc/ExampleSection'
+
+const Usage = () => (
+  <ExampleSection title="Usage">
+    <ComponentExample
+      title="Main option change"
+      description="A split button can have its main option changed based on last selection."
```

Comment:
```suggestion
      description="A SplitButton can have its main option changed based on last selection."
```

nit: to be consistent between sections